### ISSUE TITLE
Can't login because of some symbols

### ISSUE THREAD

**Issue by GoodGuyNick** [meganz/MEGAcmd#75]
Ubuntu 16.04.5. Using Bash.
MEGA CMD version: 0.9.9: code 90900

Here are my attempts:
```
~ mega-login mail@mail.com 0"6}S27:.CFS&Oo#g`-lY>bcbVt.DL[i
> mega-whoami
> ^C
~ mega-login mail@mail.com "0"6}S27:.CFS&Oo#g`-lY>bcbVt.DL[i"
> mega-whoami
> ^C
```
```
MEGA CMD> login mail@mail.com                                                                                                                                      
Password:                                                                                                                                                              
[err: 12:19:36] Login failed: invalid email or password
MEGA CMD> login mail@mail.com "0"6}S27:.CFS&Oo#g`-lY>bcbVt.DL[i"                                                                                                   
[err: 12:22:57] Login failed: invalid email or password                                                                                                                
MEGA CMD> login mail@mail.com 0"6}S27:.CFS&Oo#g`-lY>bcbVt.DL[i                                                                                                     
[err: 12:23:29] Login failed: invalid email or password 
```
I was able to login after I've changed to a more simpler password.

**Comment by polmr** [comment]
I must admit I like your password. In bash you'd need to escape all special bash characters (`"'{}[]...`) so that they are passed to the command you are executing as you expect them to (in this case mega-login). Surrounding the password between `'` and `'` should probably do. However given your attempts in the interactive shell there seems to be an issue with the way mega-cmd digests the parameters you provide. We will try to reproduce and fix it. Thanks for reporting

**Comment by GoodGuyNick** [comment]
To generate password like that I was using KeePass [password generator](https://keepass.info/help/base/pwgenerator.html)(it can help you in testing). Thank you for a great tool.

**Comment by polmr** [comment]
A better support for odd characters has been included in the new release `1.0.0` (Available at https://mega.nz/cmd). Password such as the one mentioned here should be working fine now.
